### PR TITLE
Adds delegate to set height of search results

### LINKED ISSUE #627: Searchable Table Screen With Custom Cell Type Uses Different Cell Height
I have specified a custom cell class:

```

class EventsScreen < PM::TableScreen
  title "Events"
  searchable placeholder: "Search events"
  row_height 62
  stylesheet EventsScreenStylesheet # rmq stuff

  # other stuff

  def table_data

    @events ||= Event.all
    [
      {
        cells: @events.map do |event|
          {
            properties: {
              event_name: event.event_name,
              event_details: date_time_label(event),
              event_attendees: event.number_of_attendees_in_words
            },
            search_text: "#{event.event_name} #{event.event_location}",
            cell_class: EventsCell
          }
        end
      }
    ]
  end
```

When I do the search, it does select the correct cells but it does not construct them using the specified `cell_class`. Instead, it appears to use a default height. Here is an example:

```
(main)> rmq(EventsCell).first.get.frame.inspect
=> "#<CGRect origin=#<CGPoint x=0.0 y=106.0> size=#<CGSize width=320.0 height=62.0>>"
(main)> rmq(EventsCell).last.get.frame.inspect
=> "#<CGRect origin=#<CGPoint x=0.0 y=0.0> size=#<CGSize width=320.0 height=44.0>>"
```

The first cell is one of the original ones allocated at the requested 62 points. The last one was incorrectly allocated at 44 points.

### PR DESCRIPTION
**Pull request by squidpunch**
* Fixes #627

### CODE REVIEW

**Comment by kevinvangelder** [comment]
:+1:

**Comment by squidpunch** [comment]
I think the travis failure is unrelated since I dont think its in the area I changed at all and tests passed locally, but I can review again if thats not the case :|

**Comment by jamonholmgren** [comment]
Thanks @squidpunch !

**Comment by squidpunch** [comment]
always glad to help @jamonholmgren :+1:

**Comment by squidpunch** [comment]
You may want to consider putting an iOS < 8 versus iOS 8 implementation on your radar since `UISearchDisplayController` is deprecated too.

**Comment by jamonholmgren** [comment]
Yeah, definitely, been thinking about that one since last fall. :/

File: lib/ProMotion/table/extensions/searchable.rb [hunk 1]
```diff
@@ -57,6 +57,10 @@ def searchDisplayControllerWillBeginSearch(controller)
         self.table_view.setScrollEnabled false
         @table_search_display_controller.delegate.will_begin_search if @table_search_display_controller.delegate.respond_to? "will_begin_search"
       end
+
+      def searchDisplayController(controller, didLoadSearchResultsTableView: tableView)
+        tableView.rowHeight = self.table_view.rowHeight
+      end
```
Comment (jamonholmgren) [inline lib/ProMotion/table/extensions/searchable.rb]: @markrickert Interestingly, @squidpunch had already submitted a pull request that should have fixed that earlier issue. Not sure what's going on here.

**Pull request merged**